packages/cli/tests/subjects/preload-chunks/preact.config.js

```diff
@@ -1,6 +0,0 @@
-module.exports = config => {
-	config.optimization.splitChunks = {
-		minSize: 0,
-	};
-	return config;
-};
```

Comment:
Deleted as unnecessary. We already test that preload is working fine without additional config.